Dockerfile

```diff
@@ -1,5 +1,7 @@
+ARG BASE_TAG=5.2.0
+
 FROM gcr.io/kaggle-images/python-tensorflow-whl:1.12.0-py36 as tensorflow_whl
-FROM continuumio/anaconda3:5.2.0
+FROM continuumio/anaconda3:${BASE_TAG}
```

Comment:
Not clear how this works when `BASE_TAG` is set to `${GIT_BRANCH}-staging` (which is the case when we build an image in Jenkins from a non-`master` branch.  How would a `continuumio/anaconda3` tag with our branch's commit exist?  Can you explain that bit to me pls?

Reply:
We don't override the `BASE_TAG` when building the CPU image. Only the GPU. 

I added the option to have the same behavior for both the CPU and GPU build but we don't use it.

Another option is to fail the `build` script if the `-b` flag is set w/o the `-g` but I figured it would be simpler that way and may be useful when I try bumping up the anaconda3 image in the future (I can easily run multiple build without having to update the Dockerfile and easy to track which command built it with which version).

Reply:
LGTM.  The value follows the same semantic in both cases (the tag of the base image, as the variable name is clear about).